spec.bs

```diff
@@ -1211,6 +1209,7 @@ The <dfn attribute for="AppHistoryDestination">sameDocument</dfn> getter steps a
   To <dfn>inform app history about canceling traversals</dfn> in a [=browsing context=] |bc|:
 
   1. Let |appHistory| be |bc|'s [=browsing context/active window=]'s [=Window/app history=].
+  1. If |appHistory|'s [=AppHistory/ongoing navigation signal=] is null, then return.
```

Comment:
I'm not sure about this. We can have more than one ongoing traversal in flight (and at least in the chromium implementation, ongoing navigation signal might be null while we have one ore more ongoing traversals).

Two potential ideas:
* Assert that ongoing navigation siganl is null (i.e., we recently did the "inform app history about canceling navigation" steps), 
* Invoke "inform app history about canceling navigation" if ongoing navigation signal is non-null here to ensure it is handled before iterating over the ongoing traversals.

Reply:
Good catch. After working through a variety of options here I ended up doing something similar to your second idea, although I went further and embraced the implementation even more. Let me know what you think, and note the new text in the "Canceling navigation and traversals" section.